src/FirmwarePlugin/FirmwarePlugin.h

```diff
@@ -147,7 +147,7 @@ class FirmwarePlugin : public QObject
     virtual bool sendHomePositionToVehicle(void);
     
     /// Returns the parameter that is used to identify the default component
-    virtual QString getDefaultComponentIdParam(void) const { return QString(); }
+    virtual QString getDefaultComponentIdParam(void) const { return QString("SYS_AUTOSTART"); }
```

Comment:
This is wrong. The generic case knows nothing about the parameter set it is dealing with.